### ISSUE TITLE
Nginx 1.10 compatibility

### ISSUE THREAD

**Issue by milgner** [GUI/nginx-upstream-dynamic-servers#23]
Right now this module is unfortunately incomptabible with version 1.10 of Nginx.

```
../nginx-upstream-dynamic-servers-0.4.0/ngx_http_upstream_dynamic_servers.c: In function ‘ngx_http_upstream_dynamic_server_resolve_handler’:
../nginx-upstream-dynamic-servers-0.4.0/ngx_http_upstream_dynamic_servers.c:421:20: error: assignment from incompatible pointer type [-Werror]
```

The problem here is that they introduced a separate type [ngx_resolver_addr_t](https://github.com/nginx/nginx/blob/833d03a0a2f9fa62de0f347a08c36a787bcdd8e2/src/core/ngx_resolver.h#L61) that is incompatible to `ngx_addr_t`. I don't have any experience with the Nginx source so I'm not sure whether there are any quick conversion functions or what needs to be done here...

**Comment by wandenberg** [comment]
@milgner this was fixed on this commit [29e05c5de](https://github.com/GUI/nginx-upstream-dynamic-servers/commit/29e05c5de4d9e7042f66b99d22bef878fd761219)
Use the master branch, please.

**Comment by waitspring** [comment]
Did this commit has been published in a new release?
